app.rb

```diff
@@ -0,0 +1,17 @@
+require 'sinatra/base'
+
+class Chitter < Sinatra::Base
+
+  get '/' do
+    "Welcome to Chitter"
```

Comment:
Could this use a view? Maybe could have consistent design/template across multiple views keeping the code in the controller uncluttered